src/backend/renderer/gles/mod.rs

```diff
@@ -2434,24 +2467,21 @@ impl<'frame> GlesFrame<'frame> {
             }
         }
 
+        // if we support egl fences we should use it
+        if self.renderer.extensions.iter().any(|s| s == "GL_OES_EGL_sync") {
```

Comment:
Given this is potentially a hot path, maybe also make this a bool or at least a capability, so we don't have to do string comparisions, each time we finish a frame.

Reply:
yeah, added a capability for fencing